build.gradle.kts

```diff
@@ -116,9 +116,9 @@ val plugins = listOf(
       deps = listOf("java", "org.jetbrains.plugins.gradle", "org.jetbrains.kotlin")
    ),
    PluginDescriptor(
-      since = "242.12881.48", // this version is 2024.2
+      since = "241.17011.108", // this version is 2024.2
```

Comment:
This is confusing. Did they release a 2024.2 snapshot as 241?

Reply:
No that's my bad trying to get it to work. Will fix.